webrtc.html

```diff
@@ -5078,6 +5078,42 @@ <h2>RTP Media API</h2>
       {{MediaStreamTrack}} and {{RTCRtpReceiver}} are
       surfaced to the application via the {{track}} event.
     </p>
+    <p>
+      In order for an {{RTCRtpTransceiver}} to send and/or receive media with
+      another endpoint this must be negotiated with SDP such that both endpoints
+      have an {{RTCRtpTransceiver}} object that is [= associated =] with the same
+      [= media description =].
+    </p>
+    <p>
+      When creating an offer, enough media descriptions will be generated to cover
+      all transceivers on that end. When this offer is set as the local description,
+      any unassociated transceivers get associated with media descriptions in the
```

Comment:
Sorry, one more thing I just caught. The word "unassociated" is new with this patch.

Prior terminology here [appears to be](https://w3c.github.io/webrtc-pc/#rtcrtptransceiver-interface) "disassociated". But then there are no uses of it as a noun (one use as a verb). Whichever word we pick we want to be consistent.

Reply:
And then there's https://english.stackexchange.com/questions/36416/dissociate-vs-disassociate

Reply:
OK so the existing language is "disassociate" and "disassociated". I will use the same terminology here since this is a bit of a tangent to what this PR is attempting to do. If we want to revisit what language to use when referring to "disassociated transceivers" we can do it separately and for the entire document rather than just this section. We can also define the word, currently the explanation for disassociated does look like a proper definition of the word but we don't "&lt;dfn&gt;" it.

PR updated changing "unassociated" to "disassociated" for now.